automation_tools/satellite6/upgrade/tools.py

```diff
@@ -37,20 +37,25 @@ def copy_ssh_key(from_host, to_host):
     :param to_host: A string. Hostname on to which the ssh-key will be copied.
 
     """
+    execute(lambda: run('mkdir -p ~/.ssh'), host=from_host)
+    # do we have privkey? generate only pubkey
     execute(lambda: run(
-        '[ ! -f ~/.ssh/id_rsa.pub ] && '
-        'ssh-keygen -f ~/.ssh/id_rsa -t rsa -N \'\'', warn_only=True),
-        host=from_host)
-    if int(execute(lambda: run('[ -f ~/.ssh/id_rsa.pub ]; echo $?'),
-                   host=from_host)[from_host]) == 0:
-        pub_key = execute(lambda: run(
-            'cat ~/.ssh/id_rsa.pub'), host=from_host)[from_host]
-        execute(lambda: run('[ ! -f ~/.ssh/authorized_keys ] && '
-                            'touch ~/.ssh/authorized_keys',
-                            warn_only=True), host=to_host)
+        '[ ! -f ~/.ssh/id_rsa ] || '
```

Comment:
What I don't get is that why looking for `~/.ssh/id_rsa` instead of `~/.ssh/id_rsa.pub`